frontend/src/scenes/error-tracking/errorTrackingLogic.ts

```diff
@@ -93,6 +99,12 @@ export const errorTrackingLogic = kea<errorTrackingLogicType>([
             },
         ],
     }),
+    selectors({
+        hasGroupActions: [
```

Comment:
since these are lazily created right now, this means no one will be able to create a group right?

Reply:
Correct, but I will enable it for our team.

We rely on the `$exception_fingerprint` property for grouping. We only create the Postgres record if one of those fingerprint based groups is merged, assigned or the status updated. So everything will continue to work outside of those features